content/posts/blog/2021-01-24_blog_InstantMessagingEsgehtnichtumdieApp

```diff
@@ -0,0 +1,62 @@
+die den Hype vermeidet und bei der man nicht mehrere Messenger-Apps installieren muss?
+
+Verschiedene Menschen haben verschiedene Überlegungen zur Beantwortung dieser Frage - manche benötigen einen effektiven Datenschutz, oder Datenhoheit, oder einfach die Möglichkeit, alle ihre Kontakte von einem Ort aus zu erreichen. Der Wechsel von einem Messaging-System zu einem anderen bedeutet jedoch oft, einige Kontakte zurückzulassen. Viele Messaging-Systeme erfordern außerdem eine Telefonnummer, was für den Datenschutz nicht gerade förderlich ist.
+
+### Souveränität der Kommunikation
+
+Wie Sie vielleicht schon vermutet haben, werde ich für XMPP plädieren. Ich denke, die erste zu treffende Entscheidung sollte nicht sein, welchen Messenger man verwendet, sondern welche zugrundeliegende Software-Technologie. Betrachten Sie zuerst die Wahl der Technologie, bevor Sie von einer Empfehlung zur nächsten springen.
+
+> XMPP ist ein offenes Protokoll, so wie HTTP für das Web ist. Es spielt keine Rolle, wie Ihre Website aussieht, jeder kann mit ihr interagieren. Dieses Prinzip ist die Idee hinter XMPP, allerdings für Instant Messaging.
+
+Bei der XMPP Standards Foundation und bei vielen anderen, die an dieser offenen Technologie beteiligt sind, sind wir der Meinung, dass XMPP als Kommunikationstechnologie eine gute Wahl ist, nicht nur im Hinblick auf den Datenschutz. XMPP gibt es seit über zwanzig Jahren, und in dieser Zeit wurde eine Menge praktischer Erfahrung gesammelt. Dazu gehören viele wichtige Indikatoren für die Wahl der Technologie und die Ermöglichung von Souveränität der Kommunikation; die wichtigen Indikatoren sind:
+
+- Dezentralisierung von Kommunikationsdiensten (Föderation)
+- Standardisierung und Erweiterbarkeit der Technologie
+- Interoperabilität
+- Innovation und den Einsatz offener Entwicklung
+- Privatsphäre und Kontrolle, auch durch den Einsatz von Ende-zu-Ende-Verschlüsselung
+
+> XMPP stellt definierte Informationen zur Verfügung, wie die Kommunikation in einem Netzwerk abgewickelt werden soll. Der Anwender entscheidet, welche Messaging-Software passt oder wo die Daten gespeichert werden sollen.
+
+Im Gegensatz zur aktuellen Situation, in der andere Ihnen ihren bevorzugten Messenger vorschlagen, haben Sie im XMPP-Universum die freie Wahl aus vielen Messengern - suchen Sie sich den aus, der Ihnen gefällt, nicht den, den andere mögen 
+bis zur nächsten Internet-Modeerscheinung. Der Unterschied ist, dass Sie unabhängig davon, welche Messaging-App Sie verwenden oder welche App Ihre Freunde benutzen, immer noch den Komfort haben, mit all Ihren Kontakten im selben Netzwerk zu sein, unabhängig von Ihrer heutigen Wahl. Ich denke, das ist eine echte Lösung.
```

Comment:
Der Satz klingt komisch. Ich würde etwas umformulieren. Z.B. "Im Gegensatz zur aktuellen Situation, in der andere Ihnen ihren bevorzugten Messenger vorschlagen, haben Sie im XMPP-Universum die freie Wahl aus vielen Messengern - suchen Sie sich den aus, der Ihnen gefällt, nicht den, den andere mögen und der bei der nächsten Internet-Modeerscheinung wieder ausgetauscht wird."